src/app/components/cards/Comment.scss

```diff
@@ -64,12 +68,16 @@
     }
   }
   .Comment__body {
-    padding: 0;
-    border: none;
+    @include themify($themes) {
+      padding: 0;
+      border: none;
+    }
   }
```

Comment:
Needed to add the theme here even if it is not really themified to create a more precise CSS selector for precedence over the non-collapsed comments